lib/puppet/type/docker_compose.rb

```diff
@@ -38,6 +38,13 @@ def refresh
 		end
 	end
 
+	newparam(:restart) do
```

Comment:
@garethr Thanks for your comment. Is [this line](https://github.com/AblionGE/garethr-docker/blob/7169f04ab27f0064362a1e9a10db9972c85fd651/lib/puppet/provider/docker_compose/ruby.rb#L66) enough for default value ?